includes/orphaned_objects.inc

```diff
@@ -0,0 +1,252 @@
+<?php
+
+/**
+ * @file
+ * A list of orphaned Islandora objects.
+ */
+
+/**
+ * Builds the management form for the Orphaned Objects module.
+ *
+ * @param array $form
+ *   An array representing a form within Drupal.
+ * @param array $form_state
+ *   An array containing the Drupal form state.
+ *
+ * @return array
+ *   An array containing the form to be rendered.
+ */
+function islandora_orphaned_objects_manage_form($form, $form_state) {
```

Comment:
It seems like most of the function names could follow a more natural language pattern such as `islandora_manage_orphaned_objects_form`.

Reply:
Is there much benefit to that? While the function names are a relic of its original incarnation as a module, I find that perhaps the current name structure makes it easier to read (since all the contextual information that you'd generally ignore if you know what you're looking at is grouped together at the start).

Reply:
It's only style, please don't consider it necessary. The benefit exists in conforming to expectations: 

I would not find it easier to read. I would find it hinting at a pre-existing module and incomplete refactor. This would make me look for conflicts. Following the more common naming convention would not prompt that reaction. I would find the filename `includes/orphaned_objects` to provide all necessary context. It is also notable that the information provided would not be reduced by removing this relic. The same words are present between `islandora_manage_orphaned_objects_form` and `islandora_orphaned_objects_manage_form`.

Reply:
OK, I'll give it a go. I'm just worried about missing a line when I go in renaming stuff..